integration_test/cases/repo.exs

```diff
@@ -953,6 +953,73 @@ defmodule Ecto.Integration.RepoTest do
     assert custom.bid == bid2
   end
 
+  @tag :ph
```

Comment:
Please replace :ph by :placeholders in the tags and in the descriptions. :) Also please put all placeholders tests in a single describe block.

```
describe "placeholders" do
  @describetag :placeholders
```